acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/Eth2BlockSigningRequestUtil.java

```diff
@@ -64,8 +55,14 @@ public Eth2BlockSigningRequestUtil(final SpecMilestone specMilestone) {
   }
 
   public Eth2SigningRequestBody createBlockV2Request() {
-    final BlockRequest blockRequest = new BlockRequest(specMilestone, getBeaconBlock());
+    final BlockRequest blockRequest =
+        specMilestone.isGreaterThanOrEqualTo(SpecMilestone.BELLATRIX)
```

Comment:
Wondering if we should explicitly define what milestone we are targeting so it's not hidden in the test

Reply:
done.